app/main.py

```diff
@@ -2,0 +1,20 @@
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The error message `All friends should be vaccinated` is not descriptive enough. According to the checklist, you should use descriptive error messages. It would be better to indicate which friend is not vaccinated. For example: `"{friend['name']} is not vaccinated"`.